libs/small-steps-test/small-steps-test.cabal

```diff
@@ -30,7 +30,7 @@ library
         -Wincomplete-uni-patterns -Wredundant-constraints -Wunused-packages
 
     build-depends:
-        base >=4.14 && <4.19,
+        base >=4.14 && <5,
```

Comment:
Change like this requires a patch version update. Could you please bump up the version of `small-steps-test` in the cabal file from `1.0.0.1` to `1.0.0.2`.

Also could I ask you to fix the changelog, it looks totally wrong right now:

```diff
-# Version history for `small-steps-test-test`
+# Version history for `small-steps-test`

-## 1.2.0.3
+## 1.0.0.2

-*
+* GHC-9.8 compatibility

-## 1.2.0.2
+## 1.0.0.1

-* GHC-9.6 ompatibility
+* GHC-9.6 compatibility
```

Reply:
Fixed.